packages/client/hmi-client/src/page/data-explorer/DataExplorer.vue

```diff
@@ -22,25 +22,18 @@
 					<nav>
 						<SelectButton
 							:model-value="assetType"
-							@change="if ($event.value) assetType = $event.value;"
+							@change="if ($event.value) changeAssetType($event.value);"
```

Comment:
Is the following possible?

```suggestion
							@change="changeAssetType"
```